File: Geometri C++/Triangle.cpp
#include "Triangle.h"
#include <math.h>

Triangle::Triangle(double a, double b) : Shape(a, b)
{
	c = sqrt(pow(a, 2) + pow(b, 2));
}

void Triangle::SetC(double c)
{
	this->c = c;
}

double Triangle::GetC()
{
	return c;
}

double Triangle::GetArea()
{
	return 0.5 * a * b;
}

double Triangle::GetCircumference()
{
	return a + b + c;
}

File: Geometri 03032020/Geometri/Triangle.cs
using System;

class Triangle : Shape
{
    private double c;
    public double C
    {
        get { return c; }
        set { c = value; }
    }

    public Triangle(double a, double b) : base(a, b)
    {
        c = Math.Sqrt(Math.Pow(a, 2) + Math.Pow(b, 2));
    }

    public override double GetArea()
    {
        return 0.5 * a * b;
    }

    public override double GetCircumference()
    {
        return a + b + c;
    }
}